language/move-prover/tests/README.md

```diff
@@ -88,26 +88,8 @@ MVP_TEST_FLAGS="-T=20" cargo test -p move-prover
 
 If the flag `--check-inconsistency` is given, the prover not only verifies a target, but also checks if there is any
 inconsistent assumption in the verification. If the environment variable `MVP_TEST_INCONSISTENCY=1` is set, `cargo test`
-will perform the inconsistency check while running the tests in `../../../diem-move/diem-framework`
-and `../../move-stdlib` (i.e., the prover will run those tests with the flag `--check-inconsistency`).
+will perform the inconsistency check while running the tests in `sources` (i.e., the prover will run those tests with the flag `--check-inconsistency`).
 
 ```shell script
 MVP_TEST_INCONSISTENCY=1 cargo test -p move-prover
 ```
-
-## Code coverage
-
```

Comment:
I removed this section since it seems to be talking about code coverage in diem repo and may have gotten irrelevant. I might be wrong. @junkil-park @wrwg please confirm if this is still relevant. Thanks!